test/Modules/MatrixGroupExperimental-test.jl

```diff
@@ -0,0 +1,10 @@
+@testset "Modules" begin
```

Comment:
This file should be in `test/Experimental` and the testset should have a different name, e.g.
```suggestion
@testset "GAP matrix group experiments" begin
```